### PR TITLE
Update images

### PR DESCRIPTION
**Pull request by gracshan**
### Summary 
This pull request updates the image files in Firebase storage. When a member is deleted, their image is also removed from the database.

### Test Plan 
Before:
Screen Shot 2021-05-09 at 4 35 07 PM
Screen Shot 2021-05-09 at 4 35 44 PM
After member is deleted:
Screen Shot 2021-05-09 at 4 37 39 PM

### CODE REVIEW

File: backend/src/memberAPI.ts [hunk 1]
```diff
@@ -82,7 +84,23 @@ export const deleteMember = async (
...
+  // Delete the file
+  imageFile
+    .delete()
+    .then(() => {
+      // File deleted successfully
+    })
+    .catch((error) => {
+      // Uh-oh, an error occurred!
+    });
 };
```
Comment (henry-li-06) [inline backend/src/memberAPI.ts:104]: don't think this is necesary

File: backend/src/memberAPI.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
+import { getNetIDFromEmail} from './util';
```
Comment (henry-li-06) [inline backend/src/memberAPI.ts:1]: this should be put together with the other import from `./util`

**Review by SamChou19815 — APPROVED** [review]
Nit

**Review by gracshan — COMMENTED** [review]
@SamChou19815 Can I ask if this is what you meant?

**Comment by SamChou19815** [comment]
> @SamChou19815 Can I ask if this is what you meant?

@gracshan yes that looks good

**Comment by gracshan** [comment]
> > @SamChou19815 Can I ask if this is what you meant?
> 
> @gracshan yes that looks good

@SamChou19815 Thank you!

**Pull request merged**